src/vivarium_public_health/population/mortality.py

```diff
@@ -74,6 +74,8 @@ def metrics(self, index, metrics):
         metrics['total_population__dead'] = len(the_dead)
 
         for (condition, count) in pd.value_counts(the_dead.cause_of_death).to_dict().items():
+            if condition == "death_due_to_other_causes":
```

Comment:
Why the change?

Reply:
so this is another change I was going to filter out after the fact. I got annoyed that the simulation summary at the end would say "deaths_due_to_death_due_to_other_causes." Now it just says "deaths_due_to_other_causes"

Reply:
Sweet!

Reply:
Probably not the right place to fix this though